presto-main/src/test/java/com/facebook/presto/sql/planner/TestInterpretedPageProjectionFunction.java

```diff
@@ -99,7 +99,7 @@ public void testArithmeticExpression()
     @Test
     public void testArithmeticExpressionWithNulls()
     {
-        for (ArithmeticBinaryExpression.Type type : ArithmeticBinaryExpression.Type.values()) {
+        for (ArithmeticBinaryExpression.Operator type : ArithmeticBinaryExpression.Operator.values()) {
```

Comment:
for variable should probably be called `operator` not not a `type`. Here and in other places.